modules/prediction/container/obstacles/obstacles_container.cc

```diff
@@ -344,10 +344,14 @@ void ObstaclesContainer::BuildLaneGraph() {
       AERROR << "Null obstacle found.";
       continue;
     }
-    ADEBUG << "Building Lane Graph.";
-    obstacle_ptr->BuildLaneGraph();
-    ADEBUG << "Building ordered Lane Graph.";
-    // obstacle_ptr->BuildLaneGraphFromLeftToRight();
+    if (FLAGS_prediction_offline_mode != 1 &&
+        FLAGS_prediction_offline_mode != 2) {
```

Comment:
Suggestion: Name the mode somewhere and use the names everywhere.
`constexpr int kXyzMode = 1;`
`if (FLAGS_prediction_offline_mode != kXyzMode)`